packages/react/src/tokens/themes/theme.ts

```diff
@@ -5,6 +5,463 @@ export const spaceUnit = 1;
 const fontSize = 1;
 const fontFamily = 'neue-haas-grotesk-display';
 
+const xxlXLlg = {
```

Comment:
Maybe rename this to 'smallDesktop' for better readability, future maintenance, and easier understanding for others.